### PR TITLE
Add cupy.round

### PR DESCRIPTION
**Pull request by grlee77**
NumPy has a `round` function that is equivalent to `around`, but CuPy currently does not define this function.

When checking the proposed Array API Standard, I see that `round` rather than `around` is specified in the standard, so it would be good to support it here. See:
https://data-apis.github.io/array-api/latest/API_specification/elementwise_functions.html#round-x

### CODE REVIEW

File: cupy/_math/rounding.py [hunk 1]
```diff
@@ -29,6 +29,11 @@ def round_(a, decimals=0, out=None):
     return around(a, decimals, out=out)
 
 
+round = around
+round.__name__ = 'round'
+round.__doc__ = around.__doc__.replace('around', 'round')
+
+
```
Comment (asi1024) [inline cupy/_math/rounding.py:36]: In NumPy implementation, `numpy.round` is just an alias of `numpy.round_` How about defining `round = round_` here?

Comment (grlee77) [reply cupy/_math/rounding.py:36]: Do you mean just `round = round_`, removing the `__doc__` setting? NumPy's `round` does have a docstring that is minimal and just refers the user to `around`.

Comment (grlee77) [reply cupy/_math/rounding.py:36]: Can you make a suggestion here with exactly what you want and I can accept it?

File: docs/source/reference/math.rst [hunk 2]
```diff
@@ -49,6 +49,7 @@ Rounding
 
    cupy.around
    cupy.round_
+   cupy.round
```
Comment (kmaehashi) [inline docs/source/reference/math.rst:52]: Conventionally, functions undocumented (ie, not listed in the routines list) in NumPy are left undocumented in CuPy.

File: cupy/_math/rounding.py (hunk 1 above)
Comment (kmaehashi) [reply cupy/_math/rounding.py:36]: In CuPy, undocumented aliases are defined like this:

https://github.com/cupy/cupy/blob/v9.0.0b1/cupy/__init__.py#L708-L711

So just adding `from cupy._math.rounding import round_ as round` to `cupy/__init__.py` sounds enough to me.

File: docs/source/reference/math.rst (hunk 2 above)
Comment (asi1024) [reply docs/source/reference/math.rst:52]: @grlee77 Could you address this comment?

Comment (grlee77) [reply docs/source/reference/math.rst]: I have removed it now.

**Comment by asi1024** [comment]
Jenkins, test this please.

**Review by asi1024 — APPROVED** [review]

**Comment by asi1024** [comment]
LGTM! Thanks!

**Pull request merged**